client/src/components/Common/Card/index.js

```diff
@@ -1,11 +1,9 @@
 import React from "react";
 import styled from "styled-components";
-import { spacing16 } from "../../../resources/style-constants";
 
 const Wrapper = styled.div`
   z-index: 3;
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
-  padding: ${spacing16};
```

Comment:
I removed this padding from here to make the card usable in different contexts. I fixed the usages before making this change.